cdc/model/changefeed.go

```diff
@@ -453,3 +453,40 @@ func (info *ChangeFeedInfo) fixMemoryQuota() {
 func (info *ChangeFeedInfo) fixScheduler() {
 	info.Config.FixScheduler()
 }
+
+type Barrier struct {
+	GlobalBarrierTs   Ts             `json:"global-barrier-ts"`
+	TableBarrier      []TableBarrier `json:"table-barrier"`
+	MinTableBarrierTs Ts             `json:"min-table-barrier-ts"`
+}
+
+type TableBarrier struct {
+	ID        TableID `json:"id"`
+	BarrierTs Ts      `json:"barrier-ts"`
+}
+
+func NewBarrier(ts Ts) *Barrier {
+	return &Barrier{
+		GlobalBarrierTs: ts,
+	}
+}
+
+type DownStreamType int
+
+const (
+	// DB is the type of Database.
+	DB DownStreamType = iota
+	// Other is the type of MQ or Cloud Storage.
+	Other
```

Comment:
Better to reuse the definitions in [sink pkg](https://github.com/pingcap/tiflow/blob/master/pkg/sink/sink_type.go#L74).

Reply:
This is a bit complicated. I prefer keeping things simple, so I don't want to pass sink-uri into newDDLManager().